### PR TITLE
4631 fixing tooltip in emoticons

### PR DESCRIPTION
**Pull request by NicolasBonet**
Signed-off-by: Nicolas Bonet <nibogo2@gmail.com>

### CODE REVIEW

File: Sources/Subs-Editor.php [hunk 1]
```diff
@@ -2062,8 +2063,10 @@ function create_control_richedit($editorOptions)
 			$emptyPlaceholder = 0;
 			foreach ($smileyRows as $smileyRow)
 			{
-				foreach ($smileyRow['smileys'] as $smiley)
+				foreach ($smileyRow['smileys'] as $smiley) {
```
Comment (albertlast) [inline Sources/Subs-Editor.php:2066]: in php (not js) start every brace a new line -> 
```
foreach ($smileyRow['smileys'] as $smiley) {
```
```
foreach ($smileyRow['smileys'] as $smiley)
{
```

**Comment by Sesquipedalian** [comment]
Sorry, @NicolasBonet, but the commits in this PR were also not signed-off correctly. Please create a new PR to reimplement these changes, but this time use `git commit -s` whenever you make a commit. We very much appreciate your contributions and look forward to merging them, but we do need them to be signed off in order to do so. Them's just [the rules](https://github.com/SimpleMachines/SMF2.1#how-to-contribute). :)

**Comment by albertlast** [comment]
@Sesquipedalian in my eyes was to close this pr a unnecessary move,
he can change the commit message by using rebase...
``git rebase -i HEAD~3`` -> add in the commit message the signed-off ``git push --force`` done.

**Comment by Sesquipedalian** [comment]
In my experience, GitHub handles things reasonably well if one uses that method to rewrite commits that have not yet been put into a pull request. But rewriting commits that have already been put into a pull request can have weird results. It is always cleaner to put the rewritten commits into a new pull request.

**Pull request closed without merging**